openlibrary/templates/login.html

```diff
@@ -135,7 +135,7 @@
         </div>
         <div class="formElement">
             <div class="label"></div>
-            <div class="input smaller">$:_('<a href="/account/password/forgot">Forgot Password</a>?')
+            <div class="input smaller">$:_('<a href="https://archive.org/account/login.forgotpw.php">Forgot Archive.org Password</a>?')
```

Comment:
We decided to use Open Library forget password for now, as you've recommended, and on August 1st, we will only support the IA forgot password link.

This means anyone post August 1st who has not linked and does not remember their OL password will need to contact customer support.